lib/tmp.js

```diff
@@ -547,6 +547,24 @@ function _safely_install_listener() {
     }
   }
 
+  // windows does not support signals
+  // it'd never had won if it wasn't a major PITA
+  // with node v8.x and win 10 this is no longer an issue
+  if (process.platform == 'win32') {
+    var rl = require("readline").createInterface({
```

Comment:
Please use apostrophes instead of quotes.

Reply:
done.